bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "timeshare",
+  "version": "0.0.0",
+  "homepage": "https://github.com/gdomorski/timeshare",
+  "authors": [
+    "Greg Domorski <greg.domorski@gmail.com>"
+  ],
```

Comment:
can you add another index in this array that has my info in it
should look like this 
```
"authors": [
  "Greg Domorski <greg.domorski@gmail.com>",
  "Mason Hargrove <maseh87@gmail.com>"
]
```